src/main/java/com/github/webicitybrowser/spec/fetch/FetchResponse.java

```diff
@@ -1,5 +1,7 @@
 package com.github.webicitybrowser.spec.fetch;
 
 public interface FetchResponse {
+
+	byte[] body();
```

Comment:
I'm not sure if you were planning to do this later or just didn't notice
But section 2.2.4 says this can be a byte sequence, one of a certain set of object types, or a stream
So you will probably want to abstract this a bit